docs/operator-docs/bulk-operations.md

```diff
@@ -8,10 +8,64 @@ hide_title: true
 
 Cumulus implements bulk operations through the use of `AsyncOperations`, which are long-running processes executed on an AWS ECS cluster.
 
-## Submitting a bulk request
+## Submitting a bulk API request
 
 Bulk operations are generally submitted via the endpoint for the relevant data type, e.g. granules. For a list of supported API requests, refer to the [Cumulus API documentation](https://nasa.github.io/cumulus-api/#bulk-operations). Bulk operations are denoted with the keyword 'bulk'.
 
+## Starting bulk operations from the Cumulus dashboard
+
+### Using a Kibana query
+
+> Note: You **[must have configured your dashboard build with a KIBANAROOT environment variable](https://github.com/nasa/cumulus-dashboard#configuration)** in order for the Kibana link to render in the bulk granules modal
```

Comment:
One fundamental concern (not this PR though) is that we're writing dashboard oriented docs in the Core repository.     I think it's probably the right approach to treat them as same-ecosystem in terms of the docs, but in that case, having dash docs in the dash project seems confusing. 

This probably isn't the forum to have that conversation, but we should probably discuss it.

Reply:
Yeah, it's a fine line. Since we have "operator docs" and using Kibana queries is an operator task that desperately needed demystifying, it seemed better to err on the side of putting this in the public docs